src/kakarot/accounts/library.cairo

```diff
@@ -335,6 +362,7 @@ namespace AccountContract {
         let success = [ap - 2];
         let gas_used = [ap - 1];
 
+        let (response) = alloc();
```

Comment:
no need to copy the data anymore, just use `return_data` directly and remove the `memcpy`